test/tools/unified-spec-runner/entities.ts

```diff
@@ -395,10 +403,20 @@ export class EntitiesMap<E = Entity> extends Map<string, E> {
         map.set(entity.database.id, db);
       } else if ('collection' in entity) {
         const db = map.getEntity('db', entity.collection.database);
-        const collection = db.collection(
-          entity.collection.collectionName,
-          patchCollectionOptions(entity.collection.collectionOptions)
-        );
+        const collection = await db
+          .createCollection(
+            entity.collection.collectionName,
+            patchCollectionOptions(entity.collection.collectionOptions)
```

Comment:
`patchCollectionOptions` is currently a no-op that essentially just casts the type of the options.  Is there a reason we need this utility?  I think we could just cast it inline.

Reply:
No reason, it's a no-op for now, because we're lucky to have the JSON match our API but we should actually validate that in that helper, I can make a ticket for that TODO

Reply:
Ah I see, that makes sense.  Sounds good